docs/source/lambda.rst

```diff
@@ -16,53 +16,125 @@ Each StreamAlert cluster creates two Lambda functions:
 Concepts
 --------
 * AWS Lambda `details`_
-* AWS Lambda `faq`_
+* AWS Lambda `FAQ`_
 * AWS Lambda `pricing`_
 
 .. _details: https://docs.aws.amazon.com/lambda/latest/dg/welcome.html
 .. _faq: https://aws.amazon.com/lambda/faqs/
 .. _pricing: https://aws.amazon.com/lambda/pricing/
 
-Fields
-------
-
-The following configuration settings are defined in ``variables.json``
-
-Both the ```rule_processor`` and ``alert_processor`` have the same settings.
-
-Lambda Processor Settings
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code-block::
-
-  "rule_processor_config": {
-      "handler": "stream_alert.rule_processor.main.handler",
-      "source_bucket": "prefix.streamalert.source",
-      "source_current_hash": "auto_generated_hash",
-      "source_object_key": "auto_generated_s3_object_key",
-      "third_party_libraries": [
-          "jsonpath_rw",
-          "netaddr"
-      ]
-  },
-  "rule_processor_lambda_config": {
-      "prod": [                    # Cluster name
-          10,                      # Lambda timeout
-          128                      # Lambda memory
-      ]
+
+Per Cluster Lambda Settings
+---------------------------
+
+``stream_alert`` is the main configuration module which manages:
+
+1) Lambda Functions for rule and alert processing
+2) IAM roles and policies
+3) SNS topic for alerts
+
+Template::
+
+  # conf/clusters/cluster-name.json
+
+  "stream_alert": {
+    "alert_processor": {
+      "timeout": 25,
+      "memory": 128,
+      "current_version": "$LATEST",
+      "outputs": {
+        "aws-s3": [],
+        "aws-lambda": []
+      }
+    },
+    "rule_processor": {
+      "timeout": 10,
+      "memory": 256,
+      "current_version": "$LATEST"
+    }
+  }
+
+**options**
+
+=============        ========     ======================
+Key                  Required     Description
+-------------        ---------    ----------
+``timeout``          ``true``     The time (in seconds) the Lambda function is allowed to process an incoming record. The timeout can be set to any value between 1 and 300 seconds.
+``memory``           ``true``     The amount of memory allocated for the Lambda function execution.
+``current_version``  ``true``     The most current published version of the Lambda function.
+``outputs``          ``true``    A collection of S3 bucket Ids or AWS Lambda function names to configure as valid outputs.  By default, ``aws-s3`` should contain the bucket created by the ``stream_alert`` module: ``prefix.cluster.streamalerts``.  Optionally, if the alert processor needs to invoke other Lambda functions from within your AWS account, specify a list of function names.
```

Comment:
Maybe it's just github displaying this wrong, but it looks like there is a space missing before `A collection...` that is throwing the alignment of these columns off.